storage/pebble/registers_test.go

```diff
@@ -209,6 +210,81 @@ func TestRegisters_Store_Versioning(t *testing.T) {
 	})
 }
 
+func TestRegisters_GetAndStoreEmptyOwner(t *testing.T) {
+	t.Parallel()
+	height := uint64(2)
+	emptyOwnerKey := flow.RegisterID{Owner: "", Key: "uuid"}
+	zeroOwnerKey := flow.RegisterID{Owner: flow.EmptyAddress.Hex(), Key: "uuid"}
+	expectedValue := []byte("expectedValue")
+	unexpectedValue := []byte("unexpectedValue")
+
+	t.Run("empty owner", func(t *testing.T) {
+		RunWithRegistersStorageAtInitialHeights(t, 1, 1, func(r *Registers) {
+			// First, only set the empty Owner key, and make sure the empty value is available,
+			// and the zero value returns an errors
+			entries := flow.RegisterEntries{
+				{Key: emptyOwnerKey, Value: expectedValue},
+			}
+
+			err := r.Store(entries, height)
+			require.NoError(t, err)
+
+			actual, err := r.Get(emptyOwnerKey, height)
+			assert.NoError(t, err)
+			assert.Equal(t, expectedValue, actual)
+
+			actual, err = r.Get(zeroOwnerKey, height)
+			assert.Error(t, err)
+			assert.Nil(t, actual)
+
+			// Next, add the zero value, and make sure it is returned
+			entries = flow.RegisterEntries{
+				{Key: zeroOwnerKey, Value: unexpectedValue},
+			}
+
+			err = r.Store(entries, height+1)
+			require.NoError(t, err)
+
+			actual, err = r.Get(zeroOwnerKey, height+1)
+			assert.NoError(t, err)
+			assert.Equal(t, unexpectedValue, actual)
```

Comment:
isn't it kind of expected then? maybe rename to `unexpectedValue2`?

Reply:
addressed in [902f3a8](https://github.com/onflow/flow-go/pull/4942/commits/902f3a83ae22365e223a5bc7ad304edbf2d6c8ed)